server/src/main/java/fr/uga/l3miage/tp4/models/ExamEntity.java

```diff
@@ -0,0 +1,31 @@
+package fr.uga.l3miage.tp4.models;
+
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+import java.util.Set;
+
+@Entity
+public class ExamEntity {
+
+    @Id
+    private long id;
+
+
+    private LocalDateTime startDate;
+
+
+    private LocalDateTime endDate;
+
+
+    private String name;
```

Comment:
il faut bien lire l'uml
```suggestion
    @Column(unique = true)
    private String name;
```